src/main/java/gyro/google/storage/BucketIamPolicy.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.storage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.api.services.storage.model.Policy;
+import gyro.core.resource.Diffable;
+import gyro.core.resource.Output;
+import gyro.core.resource.Updatable;
+import gyro.google.Copyable;
+
+/*
+  The Bucket's IAM policy configuration.
+ */
+public class BucketIamPolicy extends Diffable implements Copyable<Policy> {
+
+    private List<BucketIamPolicyBinding> bindings;
+
+    // Read-only
+    private Integer version;
+    private String resourceId;
+
+    /**
+     * The association between the policies' role and members who may assume that role.
+     *
+     * @subresource gyro.google.storage.BucketIamPolicyBinding
+     */
+    @Updatable
+    public List<BucketIamPolicyBinding> getBindings() {
+        if (bindings == null) {
+            bindings = new ArrayList<>();
+        }
+        return bindings;
+    }
+
+    public void setBindings(List<BucketIamPolicyBinding> bindings) {
+        this.bindings = bindings;
+    }
+
+    /**
+     * The ID of the resource to which this policy belongs.
+     */
+    @Output
+    public String getResourceId() {
+        return resourceId;
+    }
+
+    public void setResourceId(String resourceId) {
+        this.resourceId = resourceId;
+    }
+
+    /**
+     * The version of this policy. This is always set to ``3``. See also `Understanding Policies <https://cloud.google.com/iam/docs/policies#versions>`_.
+     */
+    @Output
+    public Integer getVersion() {
+        return 3;
+    }
```

Comment:
Because the version is not exposed as a field in the the console and you are not really using this field, it could be just stated as a version and maybe what it means, but the copy from should set it to what ever version the policy returns.